tmp/results/SwednClustering.json

```diff
@@ -0,0 +1,17 @@
+{
+  "all": {
+    "evaluation_time": 9.65,
```

Comment:
Looking at the file names, does this mean that normal was 4x faster than the "fast" class? 🤔

Reply:
Yep in the case of Swedn it is about the same (it is the same amount of documents being embedded), the extra cost comes from the bootstrapping and fitting clusters.